src_modules/player/commands/ShuffleCommand.ts

```diff
@@ -0,0 +1,20 @@
+import { Command, CommandContext } from '../../../commands/command';
+import { PlayerModule } from '../index';
+
+export class ShuffleCommand extends Command {
+  constructor() {
+    super({
+      name: 'shuffle',
+      description: 'Shuffle the player queue',
+      module: 'player',
+    });
+  }
+
+  async execute(ctx: CommandContext) {
+    const player = await PlayerModule.get(ctx.bot, ctx.msg);
```

Comment:
This is not a valid way to get a player instance. Use the same way as in ClearCommand.ts instead, by adding a property to the command class and the instance to the constructor.